src/main/java/com/brave/playlist/fragment/PlaylistPlayerFragment.kt

```diff
@@ -245,9 +246,9 @@ class PlaylistPlayerFragment : Fragment(R.layout.fragment_playlist_player), Play
             mSelectedPlaylistItemId = it.getString(SELECTED_PLAYLIST_ITEM_ID).toString()
         }
         val intentFilter = IntentFilter()
-        intentFilter.addAction(CAST_ACTION)
+        intentFilter.addAction(activity?.packageName +CAST_ACTION)
         intentFilter.addAction(ConstantUtils.CURRENT_PLAYING_ITEM_ACTION)
```

Comment:
Can we move concatenation to `ConstantUtils`? `CURRENT_PLAYING_ITEM_ACTION` should be changed too